ports/glutin/window.rs

```diff
@@ -605,15 +607,18 @@ impl WindowMethods for Window {
     }
 
     fn set_page_title(&self, title: Option<String>) {
+        let current_url = self.current_url.borrow().clone().unwrap();
```

Comment:
@KiChjang, I added a check to see whether the current_url is set or not. Can you see if it is correct or can you propose another approach I could take. Thanks!